execution/executor-benchmark/src/lib.rs

```diff
@@ -91,10 +92,21 @@ pub fn run_benchmark<V>(
         source_dir,
         version,
     );
+
+    let start_time = Instant::now();
     generator.run_transfer(block_size, num_transfer_blocks);
     generator.drop_sender();
     pipeline.join();
 
+    let elapsed = start_time.elapsed().as_secs_f32();
+    let delta_v = db.reader.get_latest_version().unwrap() - version;
+    info!(
```

Comment:
Just to confirm, these are the same with the numbers reported for the last block?